src/types/index.ts

```diff
@@ -611,3 +614,12 @@ export interface IDefaultComposableOptions {
    */
   store: Store<any>
 }
+
+export type WebSocketChannelTypes = 'Transactions' | 'MicroBlocks' | 'KeyBlocks';
+
+// https://github.com/aeternity/ae_mdw#websocket-interface
+export interface IMiddlewareWebSocketSubscriptionMessage {
+  op: 'Subscribe' | 'Unsubscribe',
```

Comment:
I suggest to end all interface properties with semicolon like the TS docs are doing: https://www.typescriptlang.org/docs/handbook/2/objects.html#property-modifiers